src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.tHead.querySelectorAll('th');
+
+headers.forEach(header => {
+  header.addEventListener('click', () => {
+    sortTable(header);
+  });
+});
+
+function sortTable(column) {
+  const rows = Array.from(table.tBodies[0].querySelectorAll('tr'));
+
+  rows.sort((a, b) => {
+    const aValue = a.cells[column.cellIndex].textContent;
+    const bValue = b.cells[column.cellIndex].textContent;
+
+    if (!convertToNumber(aValue)) {
```

Comment:
You are calling `convertToNumber()` twice for the same `aValue` in the sort function. Instead, store the result in a variable and reuse it.